### PR TITLE
Stream output

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

### CODE REVIEW

File: src/SapiEmitter.php [hunk 1]
```diff
@@ -98,6 +98,13 @@ public function emit(ResponseInterface $response, bool $withoutBody = false): vo
             }
         }
 
+        /**
+         * Sends headers before the body.
+         * Makes a client possible to recognize the type of the body content if it is sent with a delay,
+         * for instance, for a streamed response.
+         */
+        flush();
```
Comment (vjik) [inline src/SapiEmitter.php:106]: What about `flush()` in `emitBody()` method?

Comment (vjik) [inline src/SapiEmitter.php:106]: Add tests for this. Seems it can be implemented via `ob_start()`: `flush` will be call before `ob_end_flush`.

Comment (xepozz) [reply src/SapiEmitter.php:106]: Do you mean to move it there?
Logically it sends all headers to a client. Sending headers in the "emit body" makes me confusing.

**Comment by samdark** [comment]
@xepozz is it possible to unit-test it?

**Review by samdark — APPROVED** [review]

**Pull request merged**